examples/resources/elasticstack_kibana_alerting_rule/resource_rule_action_frequency.tf

```diff
@@ -0,0 +1,38 @@
+provider "elasticstack" {
+  kibana {}
+}
+
+resource "elasticstack_kibana_alerting_rule" "example_with_action_frequency" {
+  name     = "%s"
+  consumer = "alerts"
+  params = jsonencode({
+    aggType             = "avg"
+    groupBy             = "top"
+    termSize            = 10
+    timeWindowSize      = 10
+    timeWindowUnit      = "s"
+    threshold           = [10]
+    thresholdComparator = ">"
+    index               = ["test-index"]
+    timeField           = "@timestamp"
+    aggField            = "version"
+    termField           = "name"
+  })
+  rule_type_id = ".index-threshold"
+  interval     = "1m"
+  enabled      = true
+
+  actions {
+    # Should be the id of a MS Teams connector
```

Comment:
nit: Of any connector, right?

Reply:
In this particular example the action `params` are for a ms teams connector.